batchflow/tests/tf_models_compile_test.py

```diff
@@ -1,58 +1,59 @@
-# pylint: disable=import-error, no-name-in-module
+""" Test that all TFmodels can be constructed. Some of the bigger models are omitted due to slow initialization. """
+# pylint: disable=import-error, no-name-in-module, unused-import
 # pylint: disable=redefined-outer-name
 import pytest
 
-from batchflow.models.tf import VGG16, VGG19, VGG7
-from batchflow.models.tf import LinkNet
-from batchflow.models.tf import UNet
-from batchflow.models.tf import VNet
-from batchflow.models.tf import FCN32, FCN16, FCN8
-from batchflow.models.tf import ResNet18, ResNet34, ResNet50, ResNet101, ResNet152, \
-                    ResNeXt18, ResNeXt34 #, ResNeXt50, ResNeXt101, ResNeXt152
-# from batchflow.models.tf import Inception_v1
-# from batchflow.models.tf import Inception_v3
-# from batchflow.models.tf import Inception_v4
-# from batchflow.models.tf import InceptionResNet_v2
-from batchflow.models.tf import SqueezeNet
-from batchflow.models.tf import MobileNet, MobileNet_v2, MobileNet_v3, MobileNet_v3_small
-from batchflow.models.tf import DenseNet121 #, DenseNet169, DenseNet201, DenseNet264
-from batchflow.models.tf import ResNetAttention56, ResNetAttention92
-from batchflow.models.tf import DenseNetFC56, DenseNetFC67, DenseNetFC103
-from batchflow.models.tf import RefineNet
-from batchflow.models.tf import GCN
-from batchflow.models.tf import PyramidNet18, PyramidNet34, PyramidNet50, PyramidNet101, PyramidNet152
-from batchflow.models.tf import XceptionS, Xception41, Xception64
-from batchflow.models.tf import DeepLabXS, DeepLabX8, DeepLabX16
+from batchflow.models.tf import LinkNet, UNet, VNet, \
+                                FCN8, FCN16, FCN32, \
+                                DenseNetFC56, DenseNetFC67, DenseNetFC103, \
+                                RefineNet, GCN, DeepLabXS, DeepLabX8, DeepLabX16, \
+                                VGG16, VGG19, VGG7, \
+                                ResNet18, ResNet34, ResNet50, ResNet101, ResNet152, \
+                                ResNeXt18, ResNeXt34, ResNeXt50, ResNeXt101, ResNeXt152, \
+                                Inception_v1, InceptionResNet_v2, Inception_v3, Inception_v4, \
+                                SqueezeNet, MobileNet, MobileNet_v2, MobileNet_v3, MobileNet_v3_small, \
+                                DenseNet121, DenseNet169, DenseNet201, DenseNet264, \
+                                ResNetAttention56, ResNetAttention92, \
+                                PyramidNet18, PyramidNet34, PyramidNet50, PyramidNet101, \
+                                XceptionS, Xception41, Xception64
 
 
+
+MODELS_CLF = [
+    VGG7, VGG16, VGG19,
+    ResNet18, ResNet34, # ResNet50, ResNet101, ResNet152,
+    ResNeXt18, ResNeXt34, # ResNeXt50, ResNeXt101, ResNeXt152,
+    Inception_v1,
+    # InceptionResNet_v2, Inception_v3, Inception_v4, # fail
+    SqueezeNet,
+    MobileNet, MobileNet_v2, MobileNet_v3, MobileNet_v3_small,
+    # DenseNet121,  # DenseNet169, DenseNet201, DenseNet264,
```

Comment:
I think we should uncomment at least one model of DenseNet family

Reply:
There is one: `DenseNetFC`

Reply:
But they have different base classes.